homeassistant/components/weather/darksky.py

```diff
@@ -45,13 +60,14 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     latitude = config.get(CONF_LATITUDE, hass.config.latitude)
     longitude = config.get(CONF_LONGITUDE, hass.config.longitude)
     name = config.get(CONF_NAME)
+    lang = config.get(CONF_LANGUAGE)
 
     units = config.get(CONF_UNITS)
     if not units:
         units = 'si' if hass.config.units.is_metric else 'us'
 
     dark_sky = DarkSkyData(
-        config.get(CONF_API_KEY), latitude, longitude, units)
+        config.get(CONF_API_KEY), latitude, longitude, units, lang)
```

Comment:
Since this is listed as required in the schema, you don't need `.get`: just access it directly.